### PR TITLE
Check per-user-key reverse sig

### PR DESCRIPTION
**Pull request by mlsteele**

### CODE REVIEW

File: go/libkb/id_table.go [hunk 1]
```diff
@@ -769,6 +772,53 @@ func (s *PerUserKeyChainLink) ToPerUserKey() keybase1.PerUserKey {
 	}
 }
 
+//-------------------------------------
+
+// VerifyReverseSig checks a SibkeyChainLink's reverse signature using the ComputedKeyFamily provided.
+func (s *PerUserKeyChainLink) VerifyReverseSig(_ ComputedKeyFamily) (err error) {
```
Comment (maxtaco) [inline go/libkb/id_table.go]: any way to reuse the `Sibkey` version of this?  there are very few differences so far as i can tell?  it seems like after you get the key, it's the same thing...  factor out into a helper?

Comment (maxtaco) [inline go/libkb/id_table.go]: Also, can you test that links with broken reverse sigs fail?

Comment (mlsteele) [inline go/libkb/id_table.go]: Sure, I guess that would use https://github.com/keybase/keybase-test-vectors . Is that `iced/cson -> json -> go`?

Comment (maxtaco) [inline go/libkb/id_table.go]: hmrg.  i wonder if you can do something hacky.  that's a lot of annoyance.

**Comment by mlsteele** [comment]
Factored out and punted test to ticket.

**Pull request merged**